elasticdl/python/keras/layers/sparse_embedding.py

```diff
@@ -0,0 +1,88 @@
+import tensorflow as tf
+from tensorflow.python.keras import backend as K
+from tensorflow.python.keras import initializers
+from tensorflow.python.ops import embedding_ops, math_ops
+
+
+class SparseEmbedding(tf.keras.layers.Layer):
+    """
+    Input: indexes for the embedding entries with a shape of
+      (batch_size, input_length). Input can be either dense tensor
+      or SparseTensor.
+    Output:
+      corresponding (combined) embeddings with a shape of
+      (batch_size, input_length, output_dim) if combiner is None
```

Comment:
combiner cannot be None for sparse embedding.

Reply:
May we set a default value like "mean" ? The default value of`safe_embedding_lookup_sparse`  is "mean" .

Reply:
Yes, the default value is mean